api-report/map.api.md

```diff
@@ -62,11 +67,21 @@ export interface IDirectoryDataObject {
     };
 }
 
+// @public
```

Comment:
I do not know if it's useful to provide previous value for delete operation. Maybe? I'd add it if/when there are requests to do so, and we have good understanding of the scenario.